modules/integration_aws-backup/conf/00-aws-backup-failed.yaml

```diff
@@ -0,0 +1,22 @@
+module: AWS Backup
+name: failed
+id: backup_failed
+
+transformation: ".min(over='1h')"
+aggregation: true
+filtering: "filter('namespace', 'AWS/Backup')"
+value_unit: "count"
+
+signals:
+  failed:
+    metric: NumberOfBackupJobsFailed
+    extrapolation: zero
+    rollup: sum
+  signal:
+    formula: failed
+rules:
+  critical:
+    threshold: 0
+    comparator: ">"
+    lasting_duration: 2h
```

Comment:
lasting of 2 hours with a transformation min to 1h == 3h delay, is it really what you want ?

I think you should prefer one of the methods but avoid to use both of them in the same time.

Reply:
Addressed: https://github.com/claranet/terraform-signalfx-detectors/pull/457/files#diff-0eec7b99d6548a77ff0d378c3db0de6cae5c0527a186aeab8ed197d0e80904a3R16